### ISSUE TITLE
Home Icon after reboot with last changed activity in harmony app

### ISSUE THREAD

**Issue by Phoenix-DH** [nicoduj/homebridge-harmony#143]
I rebooted my brisge today after a long period of time. I changed in the meanwhile one activity in harmony app. I changed and renamed it.

Now i have the following icon in home:
![449EEEA3-5293-45FC-A478-5FAA77BD12C5](https://user-images.githubusercontent.com/44058592/61185916-09524000-a65f-11e9-955b-035a12f0f76c.jpeg)

How can I get rid of it. I didnt changed anything else.

**Comment by senseotech** [comment]
I've also noticed this; the device in the Home app was showing its name as one of the inputs a few weeks back.  I could still turn the device off and on and switch inputs, but I couldn't actually rename it.  I removed and re-added the homebridge instance its running on and this time I got another random activity as the name and the same behavior of being unable to rename.  I had originally chalked it up to being on the iOS13 beta, but this person appears to be on a prior version and its still happening for them.

**Comment by Phoenix-DH** [comment]
Mine has gone now ... without rebooting homebridge or changing something ...
It seems to me that this will only happen when rebooting the bridge and having something active/powered on?

**Comment by nicoduj** [comment]
Hi, the name in the home app is what you can call siri name. It can be modified inside the home app and then link to the device id. So if you want the default name of the activity you have to clean homebridge cache (or name it with the same name manually). The same is happening for inputs of tv mode, but there there is a custom file that keeps the name at the root directory of your homebridge. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by nicoduj** [comment]
Hi again, After thinking a bit more of it I think it is an input activity which is keeped by HomeKit cache and not sync since it is not recovered at restart (because not here anymore) and this not linked to the tv service . Are you using 1.0.0 or 0.7 ? I am not sure if it is linked to the homebridge cache (only there in 1.0) or not

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by senseotech** [comment]
I'm on .7 of the plugin.  Will bumping up to the 1.0 RC resolve the issue?  As it was, I had removed the TV from homekit because the issue made it unusable, so I can upgrade to the RC and re-add it relatively easy to try it out.

**Comment by nicoduj** [comment]
>I'm on .7 of the plugin.  Will bumping up to the 1.0 RC resolve the issue?  As it was, I had removed the TV from homekit because the issue made it unusable, so I can upgrade to the RC and re-add it relatively easy to try it out.

@senseotech not sure 1.0 will solve this but it could. However before trying can you post your conf and log at startup in debug mode ?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by senseotech** [comment]
Here's a copy of the log with debug turned on, right after the restart from turning it on.
[homebridge.log](https://github.com/nicoduj/homebridge-harmony/files/3473041/homebridge.log)

Config is pretty basic:

```
{
            "platform": "HarmonyHubWebSocket",
            "name": "Game Room TV",
            "hubIP": "192.168.1.216",
            "TVPlatformMode": true,
            "refreshTimer": 120
        }
```
I ran through adding the homebridge instance and TV back and got the same results where its trying to set the input names as the platform name and then oddly the room (so it was telling me my "TV" had two inputs, "Game Room TV" and "Default Room").

I removed that instance and installed 1.0RC2 onto another instance and configured it, then added the TV there, with everything behaving as expected.  The TV shows the two inputs properly named and everything.
